app/web/src/components/Navigation/Navigation.tsx

```diff
@@ -163,6 +256,57 @@ export default function Navigation() {
     </div>
   );
 
+  const menuItems = (
+    <div>
+      {menuDropDown(data).map(
+        ({ name, title, notificationCount, route, target }) => {
+          if (title)
+            return (
+              <MenuItem classes={{ root: classes.menuTitle }}>{name}</MenuItem>
+            );
+
+          const nameElement = <Typography noWrap>{name}</Typography>;
+          const hasNotification =
+            notificationCount !== undefined && notificationCount > 0;
+
+          return (
+            <MenuItem key={name}>
+              <a
+                href={route}
+                target={target || "_self"}
```

Comment:
```suggestion
                target={target}
```
target={undefined} is fine - that's the default